### ISSUE TITLE
respoke/asterisk Docker Hub image missing v14

### ISSUE THREAD

**Issue by emmaly** [respoke/asterisk-docker#6]
Would it be possible to start pushing v14 branch to Docker Hub for auto-build?  The README shows that `respoke/asterisk:14` is available as a base, but it doesn't actually appear to be available directly from Docker Hub.

**Comment by emmaly** [comment]
Oh, I've missed it.  I see a `14.0.2` tag on Docker Hub, but it seems to be eight months old.  There is no `14` tag and `latest` doesn't point to anything in `14`, but at Asterisk version `13.15.0` instead, per the [`Dockerfile`](https://hub.docker.com/r/respoke/asterisk/~/dockerfile/).

**Comment by chadxz** [comment]
I triggered those docker hub builds, updated latest to point to 14.x branch, and added 14.0 and 13.15 tags on docker. If you'd like the 14.x branch to be updated to 14.5, or the 13.x branch to be updated to 13.16 send me a pr and I'll merge it. Otherwise I'll knock it out when I'm at a computer again and have some time.

Thanks for pointing this out!

**Comment by chadxz** [comment]
Also added 14 tag on docker hub

**Comment by chadxz** [comment]
Will take a bit for all the builds to run

**Comment by chadxz** [comment]
Looks like there's an issue building the 14.x, I'll take a look asap

**Comment by emmaly** [comment]
`14` and `14.0` builds showed up on [Docker Hub](https://hub.docker.com/r/respoke/asterisk/builds/), but I don't see any new builds for a version like `14.0.2`.  I'm pointing at `14` for what I needed, so I think my purpose is completed, but if you expected `14.0.2` to be working, maybe it's not really done.  I assume the `14.0.x` branch gets tagged in Docker Hub as `14.0` and not `14.0.2`.  Or do you build on release tags instead of branches?

**Comment by chadxz** [comment]
yeah there's still some broken stuff about the build for 14 that i need to look at. Poking at it now while I have a few minutes.

**Comment by chadxz** [comment]
scratch that, will have to look later. busy lately!
